ynput/OpenPype
# Traypublisher: Fix plugin order

## Changelog Description
Frame range collector for traypublisher was moved to traypublisher plugins and changed order to make sure `assetEntity` is filled in `instance.data`.

## Testing notes:
1. Open TrayPublisher
2. Create `render` instance
3. Hit Publish (or Validate)
4. `Collect Frame Data From Asset Entity` should not crash
